README.md

```diff
@@ -9,13 +9,13 @@ Copy and paste the following HTML tags into your page:
 
 ```
 <!-- Stylesheet in head -->
-<link rel="stylesheet" href="https://cdn.chapman.edu/omni-nav/omni-nav.bundle.css">
+<link rel="stylesheet" href="https://rawgit.com/chapmanu/omni-nav/development/build/omni-nav.bundle.css">
```

Comment:
When the time comes, you can make the build folder into the orphan `gh-pages` branch, and then use GitHub’s superior hosting, e.g. https://chapmanu.github.io/omni-nav/omni-nav.bundle.css

More info: https://www.wired.com/story/github-ddos-memcached/